src/languages/perl.js

```diff
@@ -258,19 +261,44 @@ export default function(hljs) {
     end: /\}/
     // contains defined later
   };
-  const VAR = { variants: [
-    { begin: /\$\d/ },
-    { begin: regex.concat(
-      /[$%@](\^\w\b|#\w+(::\w+)*|\{\w+\}|\w+(::\w*)*)/,
-      // negative look-ahead tries to avoid matching patterns that are not
-      // Perl at all like $ident$, @ident@, etc.
-      `(?![A-Za-z])(?![@$%])`
-    ) },
-    {
-      begin: /[$%@][^\s\w{]/,
-      relevance: 0
-    }
-  ] };
+  const ATTR = {
+    className: 'attribute',
+    begin: /\s+:\s*\w+/,
+    end: /(\\s*\\(.*?\\))?/,
```

Comment:
What is this trying to match?  Is an end even necessary here if the `\w+` will match the whole attribute?

Reply:
First, the current version of the regex is already bogus, I changed from using single quotation mark to slashes and didn't change the escaping char from within the regex.
But, the idea here is that I was expecting to have the `attribute` class to span the entire attribute and its possible parameters, for instance, if we want a class to inherit a versioned class we can have:
```perl
class Example::Subclass :isa(Example::Base 2.345) { ... }
```
so the attribute would be the whole `:isa(Example::Base 2.345)` and not just `:isa`. So I was expecting to have something like:
```html
<span class="hljs-attribute"> :isa(Example::Base 2.345)</span> 
```
Instead of the attribute span get only `:isa` than the others be a normal "title" and a normal "number" (like I wrongly put in the markup test file :facepalm:)